src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -66,7 +66,7 @@ public function loadChoiceList($value = null)
             return $this->choiceList;
         }
 
-        return $this->choiceList = new ArrayChoiceList($this->getTimezones(), $value);
+        return $this->choiceList = new ArrayChoiceList(self::getTimezones(), $value);
```

Comment:
Technically, the previous code allowed extending the class to provide alternative implem via inheritance. Was it on purpose? Can't say right now, but that should be considered before approval.